src/server/middleware/nation-importer.js

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Ok we should test it -- I'll need a bit more time to get to that, but will try to give it a shot tomorrow.